WombatProject/Services/IBookAPIService.cs

```diff
@@ -6,15 +6,9 @@ namespace WombatLibrarianApi.Services
 {
     public interface IBookAPIService
     {
-        WombatBooksContext Context { get; }
-        List<Book> AuthorBookItems { get; set; }
-        List<Book> SearchResults { get; set; }
+        List<Book> SearchResults { get; }
+        List<Book> AuthorBookItems { get; }
         Task GetSearchResults(string searchTerm);
         Task GetAuthorBooks(string author);
```

Comment:
Method names should be GetSearchResultsAsync and GetAuthorBooksAsync

Reply:
Good point!